docs/table/operations.rst

```diff
@@ -268,27 +268,28 @@ Aggregation
 Aggregation is the process of applying a specified reduction function to the
 values within each group for each non-key column. This function must accept a
 ``numpy`` array as the first argument and return a single scalar value. Common
-function examples are `numpy.sum`, `numpy.mean`, and `numpy.std`.
+function examples are :func:`numpy.sum`, :func:`numpy.mean`, and
+:func:`numpy.std`.
 
 For the example grouped table ``obs_by_name`` from above, we compute the group
-means with the `~astropy.table.groups.TableGroups.aggregate` method::
-
-  >>> obs_mean = obs_by_name.groups.aggregate(np.mean)  # doctest: +SKIP
-  WARNING: Cannot aggregate column 'obs_date' [astropy.table.groups]
-  >>> print(obs_mean)  # doctest: +SKIP
-  name mag_b mag_v
-  ---- ----- ------
-  M101  15.0 13.725
-   M31  17.0   17.4
-   M82  15.7   15.5
-
-It seems the magnitude values were successfully averaged, but what
-about the WARNING? Since the ``obs_date`` column is a string-type
-array, the `numpy.mean` function failed and raised an exception.
-Any time this happens then `~astropy.table.groups.TableGroups.aggregate`
-will issue a warning and then drop that column from the output result. Note
-that the ``name`` column is one of the ``keys`` used to determine the grouping
-so it is automatically ignored from aggregation.
+means with the :meth:`~astropy.table.groups.TableGroups.aggregate` method::
+
+  >>> obs_mean = obs_by_name.groups.aggregate(np.mean)  # doctest: +SHOW_WARNINGS
+  AstropyUserWarning: Cannot aggregate column 'obs_date' with type '<U10'
+  >>> print(obs_mean)  # doctest: +FLOAT_CMP
+  name       mag_b              mag_v
+  ---- ------------------ ------------------
+  M101 15.000000000000002 13.725000000000001
+   M31               17.0 17.400000000000002
+   M82 15.699999999999998               15.5
+
+It seems the magnitude values were successfully averaged, but what about the
+``AstropyUserWarning``? Since the ``obs_date`` column is a string-type array,
```

Comment:
Maybe a full link?

```suggestion
:class:`~astropy.utils.exceptions.AstropyUserWarning`? Since the ``obs_date`` column is a string-type array,
```

Reply:
Implemented.